praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/service/ExcelExportService.java

```diff
@@ -0,0 +1,194 @@
+            ausbildungsSheet.getRow(i + 3).getCell(4).setCellValue(praktikumsstelle.namentlicheAnforderung());
+            ausbildungsSheet.getRow(i + 3).getCell(5).setCellValue(programmierkenntnisseSwitch(praktikumsstelle.programmierkenntnisse()));
+            ausbildungsSheet.getRow(i + 3).getCell(6).setCellValue(praktikumsstelle.projektarbeit() ? "Ja" : "Nein");
+            ausbildungsSheet.getRow(i + 3).getCell(7).setCellValue(ausbildungsjahrSwitch(praktikumsstelle.ausbildungsjahr()));
+            ausbildungsSheet.getRow(i + 3).getCell(8).setCellValue(praktikumsstelle.dringlichkeit().name());
+            ausbildungsSheet.getRow(i + 3).getCell(9).setCellValue(praktikumsstelle.ausbildungsrichtung().name());
+            ausbildungsSheet.getRow(i + 3).getCell(10).setCellValue(praktikumsstelle.planstelleVorhanden() ? "Ja" : "Nein");
+            ausbildungsSheet.getRow(i + 3).getCell(11).setCellValue(praktikumsstelle.assignedNwk().nachname());
+            ausbildungsSheet.getRow(i + 3).getCell(12).setCellValue(praktikumsstelle.assignedNwk().vorname());
+            ausbildungsSheet.getRow(i + 3).getCell(13).setCellValue(praktikumsstelle.assignedNwk().jahrgang());
+        }
+        for (int i = 0; i < assignedStudiumspraktikumsstellen.size(); i++) {
+            StudiumsPraktikumsstelleDto praktikumsstelle = assignedStudiumspraktikumsstellen.get(i);
+            studiumsSheet.getRow(i + 3).getCell(0).setCellValue(praktikumsstelle.referat().name());
+            studiumsSheet.getRow(i + 3).getCell(1).setCellValue(praktikumsstelle.dienststelle());
+            studiumsSheet.getRow(i + 3).getCell(2).setCellValue(praktikumsstelle.oertlicheAusbilder());
+            studiumsSheet.getRow(i + 3).getCell(3).setCellValue(praktikumsstelle.taetigkeiten());
+            studiumsSheet.getRow(i + 3).getCell(4).setCellValue(praktikumsstelle.namentlicheAnforderung());
+            studiumsSheet.getRow(i + 3).getCell(5).setCellValue(programmierkenntnisseSwitch(praktikumsstelle.programmierkenntnisse()));
+            studiumsSheet.getRow(i + 3).getCell(6).setCellValue(praktikumsstelle.dringlichkeit().name());
+            studiumsSheet.getRow(i + 3).getCell(7).setCellValue(studiensemesterSwitch(praktikumsstelle.studiensemester()));
+            studiumsSheet.getRow(i + 3).getCell(8).setCellValue(praktikumsstelle.studiengang().name());
+            studiumsSheet.getRow(i + 3).getCell(9).setCellValue(praktikumsstelle.planstelleVorhanden() ? "Ja" : "Nein");
+            studiumsSheet.getRow(i + 3).getCell(10).setCellValue(praktikumsstelle.assignedNwk().nachname());
+            studiumsSheet.getRow(i + 3).getCell(11).setCellValue(praktikumsstelle.assignedNwk().vorname());
+            studiumsSheet.getRow(i + 3).getCell(12).setCellValue(praktikumsstelle.assignedNwk().jahrgang());
+        }
+        return workbook;
+    }
+
+    private Pair<List<AusbildungsPraktikumsstelleDto>, List<StudiumsPraktikumsstelleDto>> sortPraktikumsstellen() {
```

Comment:
I don't believe it needs to be split it two, but I'd like some more feedback on that

Reply:
I think that splitting into two methods is not a must, but it would definitely make the code nicer and more readable. However, I would rename the method anyway, because at the moment it sounds as if the method is just a sorting function.